### DISCUSSION TITLE
How do I force pnpx to re-consider what version the `latest` tag is?

### DISCUSSION THREAD

**Discussion by beeman** [pnpm/pnpm#8924] [Q&A] [answered]
When I run this command and use the `latest` tag, it seems to stick to an older version.
 
This is before publishing:

```shell
 $ pnpx create-solana-dapp@latest foo
Packages: +46
++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 46, reused 46, downloaded 0, added 46, done
┌  create-solana-dapp 4.1.0
```

Then I published a new version to the `latest` tag but can't get `pnpx` to pick it up:

```
 $ pnpx create-solana-dapp@4.1.2 foo
Packages: +46
++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 46, reused 45, downloaded 1, added 46, done
┌  create-solana-dapp 4.1.2
```

Latest tag still resolves to `4.1.0`
```
$ pnpx create-solana-dapp@latest foo
┌  create-solana-dapp 4.1.0
```

If I check the versions it is listed there:

```
$ pnpm view create-solana-dapp dist-tags
{ latest: '4.1.2', next: '4.1.1' }
```

Any idea what I need to do to make `pnpx` use the actual latest?

**Comment by zkochan** [comment, marked as answer]
This will be fixed in pnpm v10. Related PR: https://github.com/pnpm/pnpm/pull/8811

**Reply by beeman** [reply]
Amazing, thanks a lot! Looking forward to the new version!
